Packs/Edgescan/Integrations/Edgescan/README.md

```diff
@@ -0,0 +1,9187 @@
+                "cves": [],
+                "cvss_score": null,
+                "cvss_v2_score": null,
+                "cvss_v2_vector": null,
+                "cvss_vector": null,
+                "cvss_version": null,
+                "date_closed": null,
+                "date_opened": "2021-02-22T13:58:30.990Z",
+                "definition_id": 300,
+                "id": 52611,
+                "label": null,
+                "last_pci_exception": null,
+                "layer": "application",
+                "location": "http://juice.edgebank.com/bank/transaction",
+                "location_specifier_id": 200,
+                "name": "Lack Of Authorization Controls",
+                "pci_compliance_status": null,
+                "risk": 4,
+                "severity": 4,
+                "status": "open",
+                "threat": 3,
+                "updated_at": "2021-06-11T08:37:55.759Z"
+            },
+            {
+                "altered_score": false,
+                "asset_id": 167,
+                "asset_name": "Edgebank - Juiceshop - Updated",
+                "confidence": null,
+                "created_at": "2021-04-02T15:35:33.749Z",
+                "cves": [
+                    "CVE-2019-8917",
+                    "CVE-2019-9546"
+                ],
+                "cvss_score": 9.8,
+                "cvss_v2_score": null,
+                "cvss_v2_vector": null,
+                "cvss_vector": "AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
+                "cvss_version": null,
+                "date_closed": null,
+                "date_opened": "2021-02-22T14:08:34.970Z",
+                "definition_id": 328,
+                "id": 52612,
+                "label": null,
+                "last_pci_exception": null,
+                "layer": "network",
+                "location": "juice.edgebank.com",
+                "location_specifier_id": 201,
+                "name": "SolarWinds Orion NPM < 12.4 RCE Vulnerability (5796)",
+                "pci_compliance_status": "fail",
+                "risk": 5,
+                "severity": 5,
+                "status": "open",
+                "threat": 5,
+                "updated_at": "2021-06-11T08:37:55.772Z"
+            }
+        ]
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Vulnerabilities
+>|id|asset_id|name|severity|cvss_score|
+>|---|---|---|---|---|
+>| 52492 | 164 | SSL Version 2 (v2) Protocol Detection | 4 | 5.3 |
+>| 52493 | 164 | SSL/TLS EXPORT_RSA <= 512-bit Cipher Suites Supported (FREAK) | 3 | 4.3 |
+>| 52494 | 164 | DNS Server Recursive Query Cache Poisoning Weakness | 3 | 5.0 |
+>| 52495 | 164 | MS17-012: Security Update for Microsoft Windows | 4 | 9.8 |
+>| 52496 | 164 | HP System Management Homepage < 6.3 Multiple Vulnerabilities | 5 | 10.0 |
+>| 52497 | 164 | Jenkins < 2.46.2 / 2.57 and Jenkins Enterprise < 1.625.24.1 / 1.651.24.1 / 2.7.24.0.1 / 2.46.2.1 Multiple Vulnerabilities | 4 | 9.8 |
```

Comment:
Same for the human readable output here. 
Let's keep only two examples.
It's hard to go over the documentation when there are too much examples

Reply:
fixed